components/page-actions.tsx

```diff
@@ -298,7 +299,9 @@ export default function PageActions({
   }
 
   function renderCancelButton() {
-    if (isWalletConnected && isBountyOpen && isBountyOwner && isBountyInDraft && !isBountyFunded)
+    const isDraftOrFunded = activeIssue?.fundingAmount.isGreaterThan(BigNumber(0)) ? !isBountyFunded : isBountyInDraft 
+    
+    if (isWalletConnected && isBountyOpen && isBountyOwner && isDraftOrFunded)
```

Comment:
`isDraftOrFunded` like we spoke yesterday, if the bounty is funded **IT CANNOT BE CANCELED**